src/index.js

```diff
@@ -19,7 +19,7 @@ const replacement = '--REDACTED--';
 
 module.exports = (whitelist = []) => {
   const terms = whitelist.join('|');
-  const paths = new RegExp(`^(${terms.replace(/\*/g, '.*')})$`, 'i');
+  const paths = new RegExp(`^(${terms.replace('.', '\\.').replace(/\*/g, '[^.]*')})$`, 'i');
```

Comment:
The `i` flag isn't doing anything here, since we're not dealing with letters. I suggest removing it.

Reply:
It is necessary here, so the path whitelist is case insensitive. There is even a [test](https://github.com/uphold/anonymizer/blob/master/test/src/index.test.js#L24-L29) for that.

Reply:
Oh, right. I misread the code and thought the `i` flag was for the `replace()`s applied to the `terms` string, rather than for the `RegExp` being constructed.